wiki/Tournaments/CWC/1/en.md

```diff
@@ -157,91 +157,322 @@ This competition has come to an end and resulted in the following podium:
+| D | **French fruit overdose club** | **2** | 0 | CatchTheHungarians | [#1](https://osu.ppy.sh/community/matches/1942894) |
+| C | **Team Nub** | **2** | 1 | Red Fury | [#1](https://osu.ppy.sh/community/matches/1950105) |
+
+## Ruleset
 
-1. Map order is already given.
-2. Participants will play at 5 maps (maximum) in Quarterfinals, 7 in Semifinals, and 9 in Finals.
+### Tournament rules
+
+1. The Catch the Beat World Cup is a 3v3, team-based tournament played on the osu!catch game mode.
+2. Beatmap scoring is based on **[ScoreV1](/wiki/Score#scorev1)**.
+3. The beatmaps for each round will be announced by the tournament host on the discussion thread in advance before the actual matches take place. Only these beatmaps will be used during the respective matches. 
+   - One beatmap will be a tiebreaker beatmap. This beatmap will only be played in case of a tie.
+4. The match schedule will be settled by the Tournament Management (see the [scheduling instructions](#scheduling-instructions)).
+5. If no staff is available, the match will be postponed.
+6. If a beatmap ends in a draw, the map will be nullified.
+7. If a player disconnects, their scores will not be counted towards their team's total.
+8. Beatmaps cannot be reused in the same match unless the map was nullified.
+9. If a team fails to bring in the required minimum of three players to a match, the corresponding match can either be declared as a forfeited match where the opposing team gets automatically awarded with a Win by Default.     
+10. Exchanging players during a match is allowed without limitations.
+    - **If a map rematch is required, exchanging players is not allowed. With the Tournament Management's discretion, an exception can be made if the previous roster is unavailable to play.**
+11. Lag is not a valid reason to nullify a beatmap.
+12. All players are supposed to keep the match running fluently and without delays. Penalties can be issued to the players if they cause excessive match delays.
+13. If a player disconnects between maps and the team cannot provide a replacement, the match can be delayed by 10 minutes at most.
+14. All players and the staff must be treated with respect. Instructions of the Tournament Management are to be followed. Decisions labeled as final are not to be objected.
+15. Disrupting the match by foul play, insulting and provoking other players or the staff, delaying the match, or other deliberate inappropriate misbehavior is strictly prohibited.
+16. Unexpected incidents are to be handled by the Tournament Management.
+17. Penalties for violating the tournament rules may include:
+    - Exclusion of specific players for one beatmap
+    - Exclusion of specific players for an entire match
+    - Declaring the match as Lost by Default
+    - Disqualification from the entire tournament
+    - Disqualification from the current and future official tournaments until appealed
+    - Any modification of these rules will be announced.
```

Comment:
this doesnt really fit here

Reply:
this is also something that is referencing the OWC 2020 page (and other modern world cup articles); i'll be keeping this fwiw

![image](https://user-images.githubusercontent.com/36564236/106346055-e04bdb80-62e6-11eb-958d-02c708c14a58.png)